### PR TITLE
Version changes for v5.14.0

### PR DESCRIPTION
**Pull request by LiamConnors**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ The instructions above apply to JupyterLab 3.x. **For JupyterLab 2 or earlier**,
 
 ```
 # JupyterLab 2.x renderer support
-jupyter labextension install jupyterlab-plotly@5.13.1 @jupyter-widgets/jupyterlab-manager
+jupyter labextension install jupyterlab-plotly@5.14.1 @jupyter-widgets/jupyterlab-manager
```
Comment (nicolaskruchten) [inline README.md:109]: this should be 5.14.0 not 5.14.1!

Comment (LiamConnors) [reply README.md]: Thanks @nicolaskruchten. Fixed it

**Comment by LiamConnors** [comment]
Encountered this issue when testing the build:
https://github.com/plotly/plotly.py/issues/4129

**Review by chriddyp — APPROVED** [review]
💃🏼

**Pull request merged**